targets/android-studio

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
+DESCRIPTION='Installs Android Studio as a directly launchable xiwi application'
+HOSTBIN='startandroidstudio'
+
+### Append to prepare.sh:
+# Must create the /etc/bash_completion.d directory
+# in order for umake to properly install
+mkdir /etc/bash_completion.d
+if [ $DISTRO != "ubuntu" ]; then
+  #Pull in the signing key
+  apt-key adv --keyserver 'keyserver.ubuntu.com' \
+                --recv-keys '07D44A5424C7C12A4BAB1E80EB563F93142986CE'
+  #Add the software source
+  cat > '/etc/apt/sources.list.d/umake.list' << EOF
+     # Bring in Umake PPA on non-Ubuntu distros
+     deb http://ppa.launchpad.net/ubuntu-desktop/ubuntu-make/ubuntu yakkety main
+     deb-src http://ppa.launchpad.net/ubuntu-desktop/ubuntu-make/ubuntu yakkety main
+  EOF
+  install ubuntu-make
```

Comment:
Since `install ubuntu-make` is always needed, can't the conditional end on line 24 with a `fi` and the `install ubuntu-make` follow?